src/main/java/pl/stormit/tinyurl/domain/model/Url.java

```diff
@@ -33,6 +33,9 @@ public class Url {
 
     private LocalDate creationDate = LocalDate.now();
 
+    @OneToOne(cascade = CascadeType.REMOVE, mappedBy = "url")
+    private UrlAnalytics urlAnalytics;
```

Comment:
Correct me if I am wrong but it shouldn't be "urls" like table name in Url class in annotation?

Reply:
No, here we refer to the url field from the UrlAnalytics class - precisely:

@OneToOne
private Url url